packages/esm-hiv-app/src/views/service-summary/encounter-list/service-summary-encounter-list.component.tsx

```diff
@@ -1,20 +1,136 @@
+import { EmptyStateComingSoon } from '@ohri/openmrs-esm-ohri-commons-lib';
 import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { EmptyStateComingSoon } from 'openmrs-esm-ohri-commons-lib';
+import {
+  EncounterTile,
+  EncounterTileColumn,
+  findObs,
+  getEncounterValues,
+  getObsFromEncounter,
+} from '../../../../../esm-commons-lib/src/components/encounter-tile/encounter-tile.component';
 
+import {
+  artTherapyDateTime_UUID,
+  art_Therapy_EncounterUUID,
+  careAndTreatmentEncounterType,
+  Cd4Count_UUID,
+  Cd4LabResultDate_UUID,
+  dateOfARtIniation,
+  enrolmentDate,
+  hivCD4Count_UUID,
+  ViralLoadResultDate_UUID,
+  ViralLoadResult_UUID,
+} from '../../../constants';
 interface OverviewListProps {
   patientUuid: string;
 }
 
 const ServiceSummaryOverviewList: React.FC<OverviewListProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
 
-  const headerTitle = t('serviceSummary', 'HIV C&T Patient Summary');
-  const displayText = t('serviceSummary', 'HIV C&T Patient Summary');
+  const headerHIV = t('hivStatus', 'HIV Status');
+  const displayTextHIV = t('hivStatus', 'HIV Status');
 
+  const headerARvRegiment = t('currentARV', 'Current ARV Regimen');
+  const displayARvRegiment = t('currentARV', 'Current ARV Regimen');
+
+  const columnsHIV: EncounterTileColumn[] = [
+    {
+      key: 'lastViralLoad',
+      header: t('vlResult', 'Last viral Load'),
+      getValue: encounter => {
+        return getEncounterValues(encounter, ViralLoadResultDate_UUID, true);
+      },
+    },
+    //TODO: Refactor to include in within VL above
+    // {
+    //   key: 'vlDate',
+    //   header: t('vlDate', 'Recent VL Date'),
+    //   getValue: encounter => {
+    //     return getObsFromEncounter(encounter, ViralLoadResult_UUID);
+    //   },
+    // },
```

Comment:
do we still need the commented code?

Reply:
Yes, this is a Todo. 
Still going to work on it today evening and close